var/spack/repos/builtin/packages/py-qtconsole/package.py

```diff
@@ -25,5 +25,6 @@ class PyQtconsole(PythonPackage):
     depends_on('py-traitlets',           type=('build', 'run'))
     depends_on('py-ipython-genutils',    type=('build', 'run'), when='@4.5.1:')
     depends_on('py-sphinx@1.3:',         type=('build', 'run'), when='+docs')
+    depends_on('py-pyqt5',               type=('build', 'run'))
```

Comment:
I don't see this dependency mentioned anywhere in the `setup.py`.

Reply:
It needs it at run time, which reminds me that I meant to change that type listing. Without it,
```console
$ jupyter-qtconsole
Traceback (most recent call last):
  File "/opt/packages/gpjohnsn/spack/opt/spack/linux-centos7-broadwell/gcc-9.1.0/py-qtconsole-4.2.1-5cied2lk6ibmeput2z5wkg5ou5lnt5hj/bin/jupyter-qtconsole", line 3, in <module>
    from qtconsole.qtconsoleapp import main
  File "/opt/packages/gpjohnsn/spack/opt/spack/linux-centos7-broadwell/gcc-9.1.0/py-qtconsole-4.2.1-5cied2lk6ibmeput2z5wkg5ou5lnt5hj/lib/python3.7/site-packages/qtconsole/qtconsoleapp.py", line 60, in <module>
    from qtconsole.qt import QtCore, QtGui
  File "/opt/packages/gpjohnsn/spack/opt/spack/linux-centos7-broadwell/gcc-9.1.0/py-qtconsole-4.2.1-5cied2lk6ibmeput2z5wkg5ou5lnt5hj/lib/python3.7/site-packages/qtconsole/qt.py", line 23, in <module>
    QtCore, QtGui, QtSvg, QT_API = load_qt(api_opts)
  File "/opt/packages/gpjohnsn/spack/opt/spack/linux-centos7-broadwell/gcc-9.1.0/py-qtconsole-4.2.1-5cied2lk6ibmeput2z5wkg5ou5lnt5hj/lib/python3.7/site-packages/qtconsole/qt_loaders.py", line 309, in load_qt
    api_options))
ImportError:
    Could not load requested Qt binding. Please ensure that
    PyQt4 >= 4.7, PyQt5 or PySide >= 1.0.3 is available,
    and only one is imported per session.

    Currently-imported Qt library:   None
    PyQt4 installed:                 False
    PyQt5 installed:                 False
    PySide >= 1.0.3 installed:       False
    Tried to load:                   ['pyqt5', 'pyside', 'pyqt']
```